src/v0/destinations/gainsight_px/util.js

```diff
@@ -1,22 +1,22 @@
 const { NetworkError } = require('@rudderstack/integrations-lib');
-const myAxios = require('../../../util/myAxios');
 const { ENDPOINTS } = require('./config');
 const tags = require('../../util/tags');
 const { getDynamicErrorType } = require('../../../adapters/utils/networkUtils');
 const { JSON_MIME_TYPE } = require('../../util/constant');
+const { handleHttpRequest } = require('../../../adapters/network');
 
 const handleErrorResponse = (error, customErrMessage, expectedErrStatus, defaultStatus = 400) => {
   let errMessage = '';
   let errorStatus = defaultStatus;
 
-  if (error.response && error.response.data) {
-    errMessage = error.response.data.externalapierror
-      ? JSON.stringify(error.response.data.externalapierror)
-      : JSON.stringify(error.response.data);
+  if (error.response) {
+    errMessage = error.response.externalapierror
```

Comment:
```suggestion
    errMessage = error?.response?.externalapierror
```

Reply:
checked error.response in if block, so it won;'t give undefined